maven-java/kalix-spring-boot-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -227,6 +227,7 @@
         <configuration>
             <systemPropertyVariables>
                 <logging.config>src/main/resources/logback-dev-mode.xml</logging.config>
+                <kalix.user-function-interface>0.0.0.0</kalix.user-function-interface>
```

Comment:
The `mvn exec:exec` is explicitly bind it to 0.0.0.0. For Spring, we should just do the same and this is how we pass `-D` properties to Spring Boot.